src/components/Image/ImageLoader.jsx

```diff
@@ -71,14 +76,23 @@ export default class ImageLoader extends Component {
     if (this.shouldLoadPrimary()) {
       this.createLoader(this.props.src)
     } else if (this.shouldLoadFallbackUrl()) {
-      getDownloadLink(this.props.photo)
+      this.getDownloadLink(this.props.photo)
         .then(url => this.setState(state => ({ ...state, fallbackUrl: url })))
         .catch(error => this.allIsLost(error))
     } else if (this.shouldLoadFallback()) {
       this.createLoader(this.state.fallbackUrl)
     }
   }
 
+  // for compatibility reasons, we try to use cozy-client but fallback on cozy-client-js
+  getDownloadLink(photo) {
+    return this.context.client
+      ? this.context.client
```

Comment:
Are we sure we want to use the context for that ? Seems like a global config, and maybe using a global is what we need.

Reply:
L'API context est utilisée pour ça dans le monde React, c'est plus ou moins une convention.

Reply:
Un composant peut être _contextualisé_ avec [momentjs](https://momentjs.com/) et un autre avec [date-fns](https://date-fns.org/) dans une même app.

D'un autre côté, les globals, même pour une app, c'est risqué, surtout si elles subissent des effets de bord.

Voilà pourquoi on _wrappe_ les composants autour de composants non UI (react-router, i18n, `<MuiThemeProvider />` de material-ui.com, etc.), afin d'y _injecter_ un contexte.